tracer/src/Datadog.Trace/IAST/Aspects/System/StringAspects.cs

```diff
@@ -272,4 +272,289 @@ public static char[] ToCharArray(string target, int startIndex, int length)
     {
         return StringModuleImpl.OnStringSubSequence(target, startIndex, target.ToCharArray(startIndex, length));
     }
+
+    /// <summary>
+    /// String.Join aspect
+    /// </summary>
+    /// <param name="separator"> sparator </param>
+    /// <param name="values"> values to join </param>
+    /// <param name="startIndex"> start index </param>
+    /// <param name="count"> number of elemnts to join </param>
+    /// <returns> Join result </returns>
+    [AspectMethodReplace("System.String::Join(System.String,System.String[],System.Int32,System.Int32)")]
+    public static string Join(string separator, string[] values, int startIndex, int count)
+    {
+        return OnStringJoin(string.Join(separator, values, startIndex, count), separator, values, startIndex, count);
+    }
+
+#if NETSTANDARD
```

Comment:
Seems these next two blocks are the same? Could you do something like:

```
#if NETSTANDARD | NETCOREAPP
```

Reply:
Yes, you are totally right... I missed that one.